Gemfile

```diff
@@ -31,8 +31,7 @@ gem 'redis', '~> 4.0'
 gem 'bootsnap', '>= 1.1.0', require: false
 
 gem 'devise'
```

Comment:
I would update devise gem to version 4.7.1 since version < 4.7.1 has potential security vulnerability .